### PR TITLE
Upgrade Pokemon Via Highest IV

### PR DESCRIPTION
**Pull request by cawk**
Will level up the pokemon with the highest IV will chose a random pokemon from the top 10 pokemon displayed at the start

also the pokedex code is in there will push that final code when i add it

### CODE REVIEW

**Comment by NecronomiconCoding** [comment]
missing translations

**Comment by cawk** [comment]
anything else that needs changing i can do this now if nothing else will need changing

**Comment by NecronomiconCoding** [comment]
doesnt seem like anything else also it should upgrade all top 10 iv pokemon

File: PoGo.NecroBot.Logic/Settings.cs [hunk 1]
```diff
@@ -584,6 +584,8 @@ public LogicSettings(GlobalSettings settings)
         public bool TransferConfigAndAuthOnUpdate => _settings.TransferConfigAndAuthOnUpdate;
         public float KeepMinIvPercentage => _settings.KeepMinIvPercentage;
         public int KeepMinCp => _settings.KeepMinCp;
+
+        public int MinPokeballsWhileSnipe => _settings.MinPokeballsToSnipe;
```
Comment (myo) [inline PoGo.NecroBot.Logic/Settings.cs]: remember to remove this xD

**Comment by cawk** [comment]
i did it as randomly select one so that it upgrades the pokemon randomly and not going through the same list over and over to make more human like
can change if you want

**Comment by cawk** [comment]
Getting conflict issues god i hate github sometimes

**Comment by myo** [comment]
same

**Pull request closed without merging**